source/index.rst

```diff
@@ -11,7 +11,184 @@
 FIRST Robotics Competition Control System
 ===========================================
 
-Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! An overview of the changes from 2020 to 2021 is available on the :ref:`New for 2021 <docs/yearly-overview/yearly-changelog:New for 2021>` document.
+Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site will contain everything you need to know for building and programming a competition robot!
```

Comment:
not sure it contains everything needed for "building" a robot but programming we do pretty good.  So maybe cut building?

Reply:
We do have a build/assembly guide.

Reply:
The subtlety being that the guide is only for assembling the kitbot, which isn't a full competition robot.